src/libumf.def

```diff
@@ -121,4 +118,7 @@ EXPORTS
     umfScalablePoolParamsSetGranularity
     umfScalablePoolParamsSetKeepAllMemory
 ; Added in UMF_0.11
+    umfFixedMemoryProviderOps
```

Comment:
Why is this change included here?

Reply:
it's an aesthethic clean up and it's related to "versioning" - if you feel like I should move this one commit to a sep. PR I can do it...

Reply:
ok, leave it here